noir/test_programs/.gitignore

```diff
@@ -1 +1,2 @@
 acir_artifacts
+execution_success/**/crs
```

Comment:
I had to add this to the git ignore after rebuilding the test programs for acir to avoid ~450 untracked files related to the crs. Maybe @kevaundray or @vezenovm can comment on whether there is some reason not to do this?